deploy-board/deploy_board/templates/clusters/replace_histories.tmpl

```diff
@@ -0,0 +1,37 @@
+{% load utils %}
+{% load static %}
+<table id="deployHistoryTableId" class="table table-condensed table-striped table-hover">
+    <tr>
+        <th>Start(-08:00)</th>
```

Comment:
Do you think it'd be better to put Active Replacement in capacity page instead of cluster configuration page?